_source/_posts/2023-05-31-semona-igama-intro.md

```diff
@@ -0,0 +1,44 @@
+---
+disqus_thread_id: 
```

Comment:
Please remove the disqus and discourse props as they aren't needed for this post